annotationPipeline/src/main/java/org/cbioportal/annotation/AnnotationPipeline.java

```diff
@@ -78,34 +78,35 @@ private static void help(Options gnuOptions, int exitStatus)
     }
 
     private static void launchJob(String[] args, String filename, String outputFilename, String outputFormat, String isoformOverride,
-            String errorReportLocation, boolean replace, Integer postIntervalSize) throws Exception
+            String errorReportLocation, boolean replace, String postIntervalSize) throws Exception
```

Comment:
@ozguzMete should this really be a string? The interval size is a number i believe. Can't immediately see where it gets parsed to integer later

Reply:
@inodb it was used like this: .addString("postIntervalSize", String.valueOf(postIntervalSize)

so, we were converting to Integer then we were converting to String again (the line above), then Spring converts it back to integer inside of MutationRecordReader class using this: 

@Value("#{jobParameters[postIntervalSize]}")
    private Integer postIntervalSize; 

tests cover this class and I manually debugged to see assigned value after your post. No problem

Reply:
ok perfect! Thank you!